src/mono/wasm/build/WasmApp.targets

```diff
@@ -486,7 +486,20 @@
 
     <Message Text="Compiling native assets with emcc. This may take a while ..." Importance="High" />
     <Exec Command='emcc $(EmccCFlags) "%(_WasmObjectsToBuild.SourcePath)" -c -o "%(_WasmObjectsToBuild.Identity)"' EnvironmentVariables="@(EmscriptenEnvVars)" />
-    <Exec Command="emcc $(EmccLDFlags) @(_DotnetJSSrcFile->'--js-library &quot;%(Identity)&quot;', ' ') @(_BitcodeFile->'&quot;%(Identity)&quot;', ' ') @(_WasmObjects->'&quot;%(Identity)&quot;', ' ') -o &quot;$(_WasmIntermediateOutputPath)dotnet.js&quot;" EnvironmentVariables="@(EmscriptenEnvVars)" />
+
+    <PropertyGroup>
+      <_EmccLinkerArguments>$(EmccLDFlags)</_EmccLinkerArguments>
+      <_EmccLinkerArguments>$(_EmccLinkerArguments) @(_DotnetJSSrcFile->'--js-library &quot;%(Identity)&quot;', ' ')</_EmccLinkerArguments>
+      <_EmccLinkerArguments>$(_EmccLinkerArguments) @(_BitcodeFile->'&quot;%(Identity)&quot;', ' ')</_EmccLinkerArguments>
```

Comment:
So the nice part of having rsp files and putting args in separate lines is that you technically don't need to quote them. Quoting cli args is hard (https://github.com/dotnet/runtime/blob/01b7e73cd378145264a7cb7a09365b41ed42b240/src/libraries/System.Private.CoreLib/src/System/PasteArguments.cs) and easy to get wrong in the ordinary case. Less so in specialized cases, but if you could avoid it, it's definitely worth looking in to.